packages/firebase_admob/README.md

```diff
@@ -1,3 +1,171 @@
+# Deprecation of firebase_admob Plugin
+The `firebase_admob` plugin will be deprecated in April 2021 in favor of [Google Mobile Ads SDK for
+Flutter](https://pub.dev/packages/google_mobile_ads).
+
+Google Mobile Ads SDK for Flutter is a new Flutter plugin that supports more Ads formats than
+`firebase_admob`. Google Mobile Ads SDK for Flutter currently supports loading and displaying
+banner, interstitial (full-screen), native ads, and rewarded video ads across AdMob and AdManager.
+It also supports displaying banner and native ads as Widgets as opposed to being overlayed over all
+app content.
+
+Projects currently using `firebase_admob` are encouraged to migrate to Google Mobile Ads SDK for
+Flutter following the instructions outlined below.
+
+## Migrating to Google Mobile Ads SDK for Flutter
+
+The main change of google_mobile_ads plugin is how ads are displayed. This section provides a quick
+migration guide when transitioning to the new plugin.
+
+### Banner and Native Ads
+
+Banner and native ads in the `firebase_admob` plugin were displayed as an overlay on top of all app
+content. With `google_mobile_ads`, they can now only be displayed as a Flutter Widget. Below is an
+example of displaying a BannerAd anchored to the bottom with an offset using the `firebase_admob`
+plugin:
+
+```dart
+void main() {
+ WidgetsFlutterBinding.ensureInitialized();
+ FirebaseAdMob.instance.initialize(appId: appId);
+
+ runApp(MyApp());
+}
+
+class MyApp extends StatefulWidget {
+ @override
+ MyAppState createState() => MyAppState();
+}
+
+class MyAppState extends State<MyApp> {
+ BannerAd _bannerAd;
+
+ @override
+ void initState() {
+   super.initState();
+   _bannerAd = BannerAd(
+     adUnitId: BannerAd.testAdUnitId,
+     size: AdSize.banner,
+   );
+
+   _bannerAd
+     ..load()
+     ..show(
+       anchorOffset: 10.0,
+       anchorType: AnchorType.bottom,
+     );
+ }
+
+ @override
+ Widget build(BuildContext context) {
+   return MyAppWidget();
+ }
+}
+```
```

Comment:
Indentation is somewhat off. single space used for tabs in some places here and below